examples/streaming/app.js

```diff
@@ -0,0 +1,32 @@
+var fs = require('fs'),
+    path = require('path'),
+    express = require('express'),
+    request = require('request'),
+    dust = require('dustjs-linkedin');
+
+dust.config.whitespace = true;
+
+dust.onLoad = function(tmpl, cb) {
+  fs.readFile(path.join('./views', tmpl + '.dust'), { encoding: 'utf8' }, cb);
```

Comment:
Yeah, I get what you're saying-- I was just trying to keep the example simple.

I can refactor slightly to resolve the tmplname before joining it to `./views`, preventing directory traversal attacks via dynamic partial names.